corelib-storage/src/main/java/eu/europeana/corelib/edm/utils/EdmUtils.java

```diff
@@ -217,9 +217,10 @@ private static void appendQualityAnnotations(RDF rdf,
         }
     }
 
+    // get the type from the main proxy
     private static String getType(FullBeanImpl fullBean) {
         for (ProxyImpl prx : fullBean.getProxies()) {
-            if (!prx.isEuropeanaProxy()) {
+            if (!prx.isEuropeanaProxy() && ProxyAggregationUtils.isProxyWithOutLineage(prx)) {
```

Comment:
If we order the proxies (see my comment in the ticket), then can just iterate over all proxies and pick the first value we find.

Reply:
done